File: joao pedro/Problema0310/Problema0310.java
import java.util.Scanner;

public class Problema0310 {
    public static void main(String[] args) {
        /*Faça um programa que leia um vetor A com dez elementos numéricos inteiros e apresente 
        o total de elementos ímpares existentes no vetor e também o percentual de números ímpares em relação 
        à quantidade total de elementos armazenados no vetor. */
        Scanner input=new Scanner(System.in);
        int vetorA[]=new int[10];
        int impar=0;
        double porcentagem;
        for(int i=0;i<10;i++){
            System.out.println("Digite um numero no vetor A de posicao "+i+": ");
            vetorA[i]=Integer.valueOf(input.nextLine());
            if(vetorA[i]%2==1){
                impar++;
            }

        }
        porcentagem=impar*0.1;
        System.out.println("Quantidade de numeros impares: "+impar);
        System.out.println("Porcentagem de numeros impares: "+porcentagem);
    }


}

File: joao pedro/Problema0310/Problema0310.py
vetorA=[]
vetor=0
impar=0
for i in range (10):
    print("Digite um numero para o vetor A de posicao ",i,": ")
    vetor=int(input(""))
    vetorA.insert(i,vetor)
    if vetor%2==1:
        impar+=1

porcentagem=impar*0.1

print("Quantidade de numeros impar: ",impar)
print("Porcentagem de numeros impar: ",porcentagem)